<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCMessageDialog</class>
 <widget class="QDialog" name="GCMessageDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gcresources.qrc">
    <normaloff>:/resources/goblinicon.jpg</normaloff>:/resources/goblinicon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
  background-color: rgba(120, 120, 120, 255);
  font: 20px;
}

QDialog
{
  background-color: rgba(150, 150, 150, 255);
}

QMessageBox
{
  background-color: rgba(120, 120, 120, 255);
}

QPushButton
{
  color: rgb(0, 0, 0);
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 240), stop:1 rgba(150, 150, 160, 255));
  border-width: 1px;
  border-radius: 5px;
  border-color: gray;
  border-style: outset;
  font: 12px;
  min-width: 5em;
  min-height: 1em;
  padding: 3px;
}

QPushButton:pressed
{
  color: rgb(190, 190, 190);
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(150, 150, 160, 240), stop:1 rgba(110, 110, 125, 255));
  border-style: inset;
}

QPushButton:default 
{
   border-color: gray;
   border-width: 2px;   
   background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(190, 190, 190, 255));
 }

QToolButton
{
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(150, 150, 160, 255));
  border-width: 1px;
  border-radius: 5px;
  border-color: rgb(40, 40, 40);
  border-style: outset;
}

QPlainTextEdit
{
   background-color: rgb(200,200,200);
   selection-background-color: rgba(120, 120, 130, 200);
   font: 12px;
}

QScrollBar:vertical 
{
   width: 17px;
   margin: 18px 0 18px 0;
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(140, 140, 140, 200), stop:1 rgba(160, 160, 160, 255));
   border-width: 1px;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::handle:vertical 
{
   background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(150, 150, 150, 255));
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/scrollgrip.png&quot;);
   min-height: 3em;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::add-line:vertical 
{
   border-color: rgba(40, 40, 40, 0);
   height: 18px;
   subcontrol-position: bottom;
   subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical 
{
   border-color: rgba(40, 40, 40, 0);
   height: 18px;
   subcontrol-position: top;
   subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/uparrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::up-arrow:vertical:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::down-arrow:vertical 
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/downarrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::down-arrow:vertical:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
   background: none;
}

QScrollBar:horizontal 
{
   height: 17px;
   margin: 0 18px 0 18px;
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(140, 140, 140, 200), stop:1 rgba(160, 160, 160, 255));
   border-width: 1px;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::handle:horizontal 
{
   background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(150, 150, 150, 255));
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/scrollgriphorizontal.png&quot;);
   min-width: 3em;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::add-line:horizontal 
{
   border-color: rgba(40, 40, 40, 0);
   width: 18px;
   subcontrol-position: right;
   subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal 
{
   border-color: rgba(40, 40, 40, 0);
   width: 18px;
   subcontrol-position: left;
   subcontrol-origin: margin;
}

QScrollBar::left-arrow:horizontal
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/leftarrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::left-arrow:horizontal:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::right-arrow:horizontal 
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/rightarrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::right-arrow:horizontal:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal 
{
   background: none;
}

QCheckBox
{
  color: rgb(0, 0, 0);
  background-color: rgba(140, 140, 140, 200);
  border-width: 1px;
  border-radius: 6px;
  padding-right: 9px;
  spacing: 10px;
  min-height: 20px;
  min-width: 20px;
  icon-size: 20px;
  font: 12px;
}

QCheckBox::indicator
{
  width: 20px;
  height: 20px;
}

QCheckBox::indicator:unchecked
{
  image: url(&quot;:/resources/checkbox.png&quot;);
}

QCheckBox::indicator:checked
{
  image: url(&quot;:/resources/checkon.png&quot;);
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="iconButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Remember my choice.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rejectButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/gcresources.qrc"/>
 </resources>
 <connections/>
</ui>
